xdmod/README.md

```diff
@@ -133,127 +155,1006 @@ If you are installing the Job Performance module (as we are in this tutorial)
+
+Settings saved.
+
+Press ENTER to continue.
+```
+
+- Press the `Enter` key to continue.
+
+## Data Warehouse Batch Export [Documentation](https://open.xdmod.org/9.5/dw-export.html#configuration)
+
+```shell
+7) Data Warehouse Batch Export
+```
+
+The Data Warehouse Batch Export feature gives users access to the raw underlying data contained within Open XDMoD's data
+warehouse. This configuration option allows for an administrator to set which directory will be used to contain the 
+exports, as well as how long those exports will be kept on disk.
+
+We'll quickly go through the configuration now
+
+- Type `7` and press the `Enter` key
+
+#### Export Directory
+The first value we're prompted to provide is the export directory. This is the directory that will be used when saving 
+the data from a user request. 
+
+
+```
+========================================================================
+Data Warehouse Batch Export
+========================================================================
+
+The data warehouse batch export feature allows users to create requests to
+export data which is then generated by a cron job and stored on the server.
+The directory where this data is stored and the duration that the data will be
+retained are configurable.
+
+Export Directory: [/var/spool/xdmod/export]
+```
+
+- Press the `Enter` key to accept the default value.
+
+#### Export File Retention Duration
+Next you will be prompted to enter the number of days these exports should be retained on disk.
+
+```
+If the export directory does not exist, it must be created and assigned the
+correct permissions and ownership.  It must be readable by the web server and
+both readable and writable by the user that is used to generate the export
+files.  By default, the web server user is expected to be apache and
+the group is expected to be xdmod.  If your system uses a different
+user and group then the automatic process will fail and you must set the
+permissions manually.
+
+Export File Retention Duration in Days: [31]
+```
+
+- Press the `Enter` key to accept the default value.
+
+You should then be prompted to save these settings in the `portal_settings.ini` file
+```shell
+Overwrite config file '/etc/xdmod/portal_settings.ini' (yes, no)? [yes]
+```
+
+- Press the `Enter` key to accept the default value.
+
+If the save is successful then you should see the following:
+```shell
+Writing configuration to '/etc/xdmod/portal_settings.ini'
+
+Settings saved.
+
+Press ENTER to continue.
+```
+
+### Automatically Check for Updates
+
+```
+8) Automatically Check for Updates
+```
+
+This is the final piece of setup that, while not strictly necessary for a fully operational Open XDMoD installation, 
+means you will be kept up to date when the newest version is released.
+
+- Type `8` and press the `Enter` key.
+
+#### Enable Automatic Update Check
+Like it says on the tin, by answering `yes` your XDMoD installation will periodically check to see if there are any 
```

Comment:
Is this true?

Reply:
Answer: sort of. XDMoD does not actually do any auto checking - instead there is a line in the crontab that runs the check command once a month. They actually have to have the cron enabled for this to work.

Reply:
*nod* gotcha, I'll clarify that